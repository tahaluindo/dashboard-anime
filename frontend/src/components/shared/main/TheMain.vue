<template>
  <main class="min-h-screen">
    <slot />
  </main>
</template>

<script lang="ts">
import { defineComponent } from "vue";

export default defineComponent({});
</script>

<style lang="postcss">
</style>