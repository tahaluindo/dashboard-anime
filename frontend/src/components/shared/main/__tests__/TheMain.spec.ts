import { mount } from "@vue/test-utils";
import TheMain from "../TheMain.vue";

describe("TheMain component test", () => {
  it("should render the default component.", () => {
    const wrapper = mount(TheMain, {
      shallow: true,
    });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
