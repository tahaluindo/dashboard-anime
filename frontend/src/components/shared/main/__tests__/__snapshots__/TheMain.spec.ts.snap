// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheMain component test should render the default component. 1`] = `"<main class=\\"min-h-screen\\"></main>"`;
